packages/devtools_app/lib/src/shared/ui/vm_flag_widgets.dart

```diff
@@ -64,33 +63,34 @@ class CpuSamplingRateDropdown extends StatelessWidget {
             screenId,
           );
         }
-        return SizedBox(
-          height: defaultButtonHeight,
-          child: DevToolsTooltip(
-            message:
-                'The frequency at which the CPU profiler will sample the call stack',
-            child: RoundedDropDownButton<String>(
-              key: CpuSamplingRateDropdown.dropdownKey,
-              isDense: true,
-              style: Theme.of(context).textTheme.bodyMedium,
-              value: safeValue,
-              items: [
-                _buildMenuItem(CpuSamplingRate.low),
-                _buildMenuItem(CpuSamplingRate.medium),
-                _buildMenuItem(CpuSamplingRate.high),
-              ],
-              onChanged: _onSamplingFrequencyChanged,
-            ),
-          ),
+        return AnalyticsDropDownButton(
+          key: CpuSamplingRateDropdown.dropdownKey,
+          gaScreen: screenId,
+          isDense: true,
+          gaDropDownId: gac.CpuProfilerEvents.profileGranularity.name,
+          message:
+              'The frequency at which the CPU profiler will sample the call stack',
+          value: safeValue,
+          items: [
+            _buildMenuItem(CpuSamplingRate.low),
+            _buildMenuItem(CpuSamplingRate.medium),
+            _buildMenuItem(CpuSamplingRate.high),
+          ],
+          onChanged: _onSamplingFrequencyChanged,
```

Comment:
we need to remove the analytics call from `_onSamplingFrequencyChanged` or else we will send analytics twice.